packages/graphql/src/translate/create-set-relationship-properties.ts

```diff
@@ -60,6 +63,22 @@ function createSetRelationshipProperties({
         }
     });
 
+    relationship.primitiveFields.forEach((field) => {
+        if (!field.callback) {
+            return;
+        }
+
+        const paramName = `${varName}_${field.fieldName}_${field.callback?.name}`;
+
+        callbackBucket.addCallback({
+            functionName: field.callback?.name,
+            paramName,
+            parent: properties,
+        });
+
+        strs.push(`SET ${varName}.${field.dbPropertyName} = $callbacks.${paramName}`);
```

Comment:
This is repeated a lot in various files, should it be a shared function?

Reply:
Moved it to a util function in https://github.com/neo4j/graphql/pull/1146/commits/9ae6ae67184430692b173e4e5708ece6855f39ee but I'm not sure this is a good place to put it..